### ISSUE TITLE
`julia-snail-send-region` and `julia-snail-send-top-level-form` not module aware when called outside of main module file

### ISSUE THREAD

**Issue by orialb** [gcv/julia-snail#17]
Hi,
Thanks for continuing to invest time in this nice package!

I think I might encountered an issue with `julia-snail-send-top-level` and `julia-snail-send-region` when used with a module which includes several files. 

Say I have the following module defined in `TestSnail.jl`:
```julia
module TestSnail

include("foo.jl")

end
```
where the contents of `foo.jl` are:
```julia
function foo()
    println("foo")
end
```

- After running `julia-snail-send-buffer-file` in `TestSnail.jl`, the `TestSnail` module is loaded and I can call `TestSnail.foo()` in the REPL. 
- I do some change in the function `foo` and try to evaluate it using either `julia-snail-send-top-level-form` or `julia-snail-send-region` (which are supposed to be module aware according to docs) 
- This seems to be evaluate the definition of `foo` in the `Main` module instead of in `TestSnail` (i.e. a new function will be defined in the main scope).

**Comment by gcv** [comment]
Should be fixed now in 9791582 and in MELPA. Please take a look.

**Comment by orialb** [comment]
I can confirm that this is fixed for me.
Thanks for the quick fix!
